manifests/config.pp

```diff
@@ -1,14 +1,14 @@
 # == Class: prosody::config
 class prosody::config {
-  file { '/etc/prosody/conf.avail':
+  file { "${prosody::config_directory}/conf.avail":
```

Comment:
Here also we need to `include prosody`.

But I think this is a private clas,s in which case we can just use `assert_private()` and bcause the code can only be included from the module, we can ski[ the include completely.